packages/veritone-json-schemas/schemas/vtn-standard/object/object.json

```diff
@@ -0,0 +1,82 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://docs.veritone.com/schemas/vtn-standard/object.json",
+  "title": "vtn-standard.object",
+  "description": "Standard engine output for Object at Veritone",
+  "type": "object",
+  "definitions": {
+    "objectResult": {
+      "$ref": "master.json#/definitions/objectResult"
+    }
+  },
+  "properties": {
+    "object": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "allOf": [
+          {
+            "properties": {
+              "object": {
+                "$ref": "#/definitions/objectResult"
+              },
+              "type": {
+                "type": "string",
+                "pattern": "object"
+              }
+            }
+          }
+        ]
+      }
+    },
+    "series": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "allOf": [
+          {
+            "properties": {
+              "object": {
+                "$ref": "#/definitions/objectResult"
+              },
+              "startTimeMs": {
+                "type": "integer",
+                "minimum": 0
+              },
+              "stopTimeMs": {
+                "type": "integer",
+                "minimum": 0
+              },
+              "type": {
+                "type": "string",
+                "pattern": "object"
+              }
+            },
+            "required": [
+              "startTimeMs",
```

Comment:
`type` should also be required for this, as should `object` (at least for this object validationContract)